spec/models/manageiq/providers/vmware/infra_manager/provision_spec.rb

```diff
@@ -135,12 +135,34 @@
           end
         end
 
+        let(:dc_nested) do
+          FactoryGirl.create(:datacenter).tap do |f|
+            f.parent = FactoryGirl.create(:ems_folder, :name => 'testing').tap { |e| e.parent = @ems; }.tap do |test|
```

Comment:
Do you need the `.tap { |e| e.parent = @ems; }` if you `.tap` and set parent again on the next line?